lib/geokit/geocoders/mapbox.rb

```diff
@@ -17,7 +17,7 @@ def self.do_reverse_geocode(latlng)
       end
 
       # Template method which does the geocode lookup.
-      def self.do_geocode(address)
+      def self.do_geocode(address, options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.